src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.card_stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:nth-child(-n + 4) {
```

Comment:
In this manner, you are fixing the styling for only 4 stars, you should use the code from the previous task _Stars_ so it can be changed as needed.
This comment just complements the previous one I made.